app/code/Magento/BraintreeGraphQl/etc/schema.graphqls

```diff
@@ -11,9 +11,9 @@ input PaymentMethodInput {
 }
 
 input BraintreeInput {
-    payment_method_nonce: String!
-    is_active_payment_token_enabler: Boolean!
-    device_data: String
+    payment_method_nonce: String! @doc(description:"Required input for Braintrees payments")
+    is_active_payment_token_enabler: Boolean! @doc(description:"Required input for Braintrees payments")
+    device_data: String @doc(description:"Additional information for Braintrees payments")
```

Comment:
The `device_data` will be presented only in a case if Kount (advanced fraud protection) is enabled for Braintree payment integration. The device data contains a fingerprint provided by Braintree JS SDK and should be sent with sale transaction details to the Braintree payment gateway.